addon/styles/_list-item-primitives.scss

```diff
@@ -0,0 +1,185 @@
+@import 'node_modules/ember-frost-core/addon/styles/frost-theme';
+
+$frost-list-item-primary-state-critical: #E52C2C;
```

Comment:
Do you have sass linting enable? I think sass linting wants this to be in small case.